org.scram_pra.scram.yml

```diff
@@ -0,0 +1,42 @@
+app-id: org.scram_pra.scram
+runtime: org.kde.Sdk
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: scram-gui
+rename-icon: scram
+rename-desktop-file: scram-gui.desktop
+finish-args:
+- "--socket=wayland"
+- "--socket=fallback-x11"
+- "--filesystem=home"
+- "--share=ipc"
+# - "--device=dri" # TODO: needed?
+modules:
+- boost.json
+# libxml2 is already in the sdk
+- name: scram
+  buildsystem: cmake-ninja
+  config-opts:
+  - -DCMAKE_BUILD_TYPE=Release
+  - -DINSTALL_PREFIX=/app
+  - -DBUILD_GUI=ON
+  - -DWITH_TCMALLOC=OFF
+  - -DWITH_JEMALLOC=OFF
+  - -DWITH_COVERAGE=OFF
+  - -DWITH_PROFILE=OFF
+  builddir: true
+  post-install:
+  - desktop-file-install --dir=/app/share/applications gui/scram-gui.desktop
```

Comment:
I this file already being installed but the build.

Also the path is relative to the builddir, hence it's not found. (build failure)

So you should just remove it.